code/lib/cli/src/automigrate/fixes/vite-config-file.ts

```diff
@@ -0,0 +1,111 @@
+import { dedent } from 'ts-dedent';
+import type { Fix } from '../types';
+import findUp from 'find-up';
+import { getFrameworkPackageName } from '../helpers/mainConfigFile';
+import { frameworkToRenderer } from '../../helpers';
+import { frameworkPackages } from '@storybook/core-common';
+
+interface Webpack5RunOptions {
+  plugins: string[];
+  existed: boolean;
+}
+
+export const viteConfigFile = {
+  id: 'viteConfigFile',
+
+  async check({ mainConfig, packageManager }) {
```

Comment:
I don't see any handling of custom vite config locations, was that a deliberate decision?

https://storybook.js.org/docs/builders/vite#override-the-default-configuration

Reply:
Was missed indeed. I didn't know this was an option.